app/views/admin/recovery_displays/_form.html.haml

```diff
@@ -3,53 +3,63 @@
   - if @recovery_display.stolen_record_id.present?
     = f.hidden_field :stolen_record_id, value: @recovery_display.stolen_record_id
-  .form-group
-    = f.label :quote_by
-    = f.text_field :quote_by, class: "form-control w-25"
-  .form-group
-    = f.label :link
-    = f.text_field :link, class: "form-control w-25"
-  - if @bike.present?
-    .form-group
-      %label
-        Bike url
-      %input.less-strong{ type: 'text', value: bike_url(@recovery_display.bike) }
-  .form-group
-    = f.label :date_recovered
-    - f.object.date_recovered ||= Time.now
-    = f.datetime_local_field :date_recovered, max: Time.now + 1.day, step: 60, required: true, class: 'form-control w-25 dateInputUpdateZone', "data-initialtime" => l(f.object.date_recovered, format: :convert_time)
-  .form-group
-    .row
-      .col-sm
-        = f.label :image, "Recovery photo"
-        - if @recovery_display.image?
-          = image_tag(@recovery_display.image_url(:thumb))
-        - else
-          %p
-            No picture!
-      .col-sm
-        = f.file_field :image, class: "float-right"
-      .col-sm
-  .form-group
-    - if @recovery_display.image.present?
-      = f.label :remove_image
-      = f.check_box :remove_image
-    - elsif @bike.present? && @recovery_display.image.blank?
-      - if @bike.public_images.present? || @bike.stock_photo_url.present?
-        - if @bike.public_images.present?
-          - text = "has image"
-          - url = @bike.public_images.first.image_url
-        - else
-          - text = "has stock image"
-          - url = @bike.stock_photo_url
-        #{link_to "Bike", bike_url(@bike), target: "_blank"} #{text}
-        %a.btn.btn-default{ href: '#', data: {url: url}}
-          Use first image
-          = f.hidden_field :remote_image_url
-      - else
-        .controls
-          Bike has no image
-    .form-group
-      = f.label :body
-      = f.text_area :quote, placeholder: 'Quote about recovery', rows: 4, class: "form-control w-50"
-    = f.submit 'Save', class: 'btn btn-success float-right mb-4'
+  .card.mb-4.bg-light
+    .card-body
+      .row
+        .col-lg-6
+          .form-group
+            = f.label :quote_by
+            = f.text_field :quote_by, class: "form-control"
+        .col-lg-6
+          .form-group
+            = f.label :link
+            = f.text_field :link, class: "form-control"
+      .row
+        .col-lg-6
+          .form-group
+            = f.label :date_recovered
+            - f.object.date_recovered ||= Time.now
+            = f.datetime_local_field :date_recovered, max: Time.now + 1.day, step: 60, required: true, class: 'form-control dateInputUpdateZone', "data-initialtime" => l(f.object.date_recovered, format: :convert_time)
+        .col-lg-6
+          - if @bike.present?
+            .form-group
+              %label
+                Bike url
+              %input.less-strong.w-100{ type: 'text', value: bike_url(@recovery_display.bike) }
+      .form-group.mt-4
```

Comment:
I think `form-group` should be nested _inside_ of the column, and the row should be the top level.